config/locales/ru/resources.yml

```diff
@@ -240,18 +238,18 @@ ru:
         url: "Сайт"
         tracks:
           one: "%{count} Трек" # Eg. 1, 21, 31, 41, 51, 61, 71, 81, 101, 1001, …
-          few: "%{count} Tracks" # Eg. 2~4, 22~24, 32~34, 42~44, 52~54, 62, 102, 1002, … # TODO: Translate this
+          few: "%{count} Трека" # Eg. 2~4, 22~24, 32~34, 42~44, 52~54, 62, 102, 1002, … 
           many: "%{count} Треков" # Eg. 0, 5~19, 100, 1000, 10000, 100000, 1000000, …
-          other: "%{count} Tracks" # Eg. 0.0~1.5, 10.0, 100.0, 1000.0, 10000.0, 100000.0, 1000000.0 # TODO: Translate this
+          
 
       author:
         name: :activerecord.attributes.generic.name # Do not translate
         country_code: "Страна"
         title: "Обращение"
         image: "Фото"
         description: "Биография"
-        years_meditating: "Опыт Медитации"
-        user: "Cвязанный с Автором Аккаунт Админа"
+        years_meditating: "Опыт медитации"
```

Comment:
This is a field title, it would make sense for it to be title case

Reply:
changed to capital letters